EIPS/eip-4353.md

```diff
@@ -0,0 +1,238 @@
+---
+eip: 4353
+title: Interface for Staked Tokens in NFTs
+description: This interface enables access to publicly viewable staking data of an NFT.
+author: Rex Creed (@aug2uag), Dane Scarborough <dane@nftapps.us>
+discussions-to: https://ethereum-magicians.org/t/eip-4353-viewing-staked-tokens-in-nft/7234
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-10-08
+---
+
+## Abstract
+ERC721s can be staked with tokens, but there's no means of retrieving the number of tokens staked and/or bound to an NFT. This proposal outlines a standard that may be implemented by all wallets and marketplaces easily to correctly retrieve the staked token amount of an NFT.
+
+## Motivation
+The absence of staked tokens data limits the ability of the token owner to convey to another, for any purpose including to transact, the actual amount of staked tokens as may be viewable in a wallet, marketplace, or explorer. The ability to identify and verify an exogenous value derived from the staking process may be critical to the aims of an NFT holder.
+
+## Specification
+```js
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @dev Interface of the ERC165 standard, as defined in the
+ * https://eips.ethereum.org/EIPS/eip-165[EIP].
+ *
+ * Implementers can declare support of contract interfaces, which can then be
+ * queried by others ({ERC165Checker}).
+ *
+ * For an implementation, see {ERC165}.
+ */
+interface IERC721Staked {
+    
+    /**
+     * @dev Returns true if this contract implements the interface defined by
+     * `interfaceId`. See the corresponding
+     * https://eips.ethereum.org/EIPS/eip-165#how-interfaces-are-identified
+     * to learn more about how these ids are created.
+     *
+     * This function call must use less than 30 000 gas.
+     */
+    function supportsInterface(bytes4 interfaceId) external view returns (bool);
+
+     /**
+     * @dev Returns uint256 amount of on-chain tokens staked to the NFT.
+     * 
+     * @dev Wallets and marketplaces would need to call this for displaying
+     *      the amount of tokens staked and/or bound to the NFT.
+     */
+    function stakedAmount(uint256 tokenId) external view returns (uint256);
+    
+}
+```
+
```

Comment:
It's generally a good idea to include the EIP-165 interface identifier somewhere in your EIP.

Reply:
is there an example you can point me to?

Reply:
updated/corrected comment and now includes `Note: The ERC-165 identifier for this interface is 0x3a3d855f.`